<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TabEtikette</class>
 <widget class="QWidget" name="TabEtikette">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>826</width>
    <height>632</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_3">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_23">
     <property name="spacing">
      <number>4</number>
     </property>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="label_8">
         <property name="text">
          <string>Vorlage:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="ComboBox" name="cbAuswahl">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="toolTip">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Über diesen Reiter werden zuvor abgespeicherte &lt;span style=&quot; font-weight:600;&quot;&gt;Gestaltungsvorlagen &lt;/span&gt;aufgerufen. Mit Vorlagen lassen sich Etiketten schnell und unkompliziert gestalten.&lt;/p&gt;&lt;p&gt;Die Gestaltungsmöglichkeiten beziehen sich dabei grundlegend auf Einträge und eingepflegte Bestände im Kleinen Brauhelfer.&lt;/p&gt;&lt;p&gt;Der Funktionsumfang ist zweckmäßig. Er erfüllt nicht die Anforderungen mächtiger Design- und Grafikprogramme. Es können aber viele moderne Designelemente mittels CSS3 und HTML5 definiert werden.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnLoeschen">
         <property name="minimumSize">
          <size>
           <width>24</width>
           <height>22</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>24</width>
           <height>22</height>
          </size>
         </property>
         <property name="icon">
          <iconset resource="images.qrc">
           <normaloff>:/images/close_button.svg</normaloff>:/images/close_button.svg</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>25</width>
           <height>16</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QLabel" name="label_7">
       <property name="text">
        <string>Weitere Vorlagen können über die Sudanhänge im Reiter Rezept hinzugefügt werden. Die Datei muss im SVG Format sein.</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSplitter" name="splitter_1">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <widget class="QWidget" name="verticalLayoutWidget">
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <property name="spacing">
          <number>4</number>
         </property>
         <item>
          <widget class="SvgView" name="viewSvg">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
             <horstretch>1</horstretch>
             <verstretch>1</verstretch>
            </sizepolicy>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="QCheckBox" name="cbEditMode">
             <property name="toolTip">
              <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Im &lt;span style=&quot; font-weight:600;&quot;&gt;Editiermodus&lt;/span&gt; ist die Bearbeitung der Gestaltungsvorgaben in der linken Spalte sowie der Tags in der rechten Spalte möglich.&lt;/p&gt;&lt;p&gt;Tags speisen Werte in das Layout ein.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
             </property>
             <property name="text">
              <string>Editiermodus</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="cbTagsErsetzen">
             <property name="toolTip">
              <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Angeklickt, können hiermit eigene &lt;span style=&quot; font-weight:600;&quot;&gt;Tags&lt;/span&gt; definiert werden.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
             </property>
             <property name="text">
              <string>Tags ersetzen</string>
             </property>
             <property name="checked">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QPushButton" name="btnSaveTemplate">
             <property name="toolTip">
              <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Hier wird das &lt;span style=&quot; font-weight:600;&quot;&gt;Etikett&lt;/span&gt; gespeichert.&lt;/p&gt;&lt;p&gt;Dies kann nicht rückgängig gemacht werden.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
             </property>
             <property name="text">
              <string>Speichern</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
       <widget class="QSplitter" name="splitter_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <widget class="QPlainTextEdit" name="tbTemplate">
         <property name="toolTip">
          <string>In dieser Spalte</string>
         </property>
         <property name="lineWrapMode">
          <enum>QPlainTextEdit::NoWrap</enum>
         </property>
        </widget>
        <widget class="QPlainTextEdit" name="tbTags">
         <property name="lineWrapMode">
          <enum>QPlainTextEdit::NoWrap</enum>
         </property>
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </widget>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>0</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_27">
     <property name="spacing">
      <number>4</number>
     </property>
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string>PDF Ausgabe</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="7" column="0">
         <spacer name="verticalSpacer_6">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="1" column="0">
         <widget class="QSpinBox" name="spinBox_BreiteLabel">
          <property name="toolTip">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Hier wird die &lt;span style=&quot; font-weight:600;&quot;&gt;Breite &lt;/span&gt;der Flaschenbinde oder des Etiketts festgelegt.&lt;/p&gt;&lt;p&gt;Die ideale Breite ist abhängig vom Umfang des Gebindes und sollte vorher ausgemessen werden.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
          <property name="maximum">
           <number>210</number>
          </property>
          <property name="value">
           <number>185</number>
          </property>
         </widget>
        </item>
        <item row="5" column="0">
         <widget class="QSpinBox" name="spinBox_AbstandLabel">
          <property name="toolTip">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Hier wird der &lt;span style=&quot; font-weight:600;&quot;&gt;Abstand&lt;/span&gt; zwischen einzelnen Etiketten festgelegt.&lt;/p&gt;&lt;p&gt;Üblich sind 2mm Beschnittzugabe, um Ungenauigkeiten wie sogenannte Blitzer zu vermeiden, wenn Farbe bis zum Rand gedruckt wird. Dabei werden 2mm Farbauftrag zum Rand addiert. Blitzer entstehen bei Ungenauigkeiten während des Papierschnitts.&lt;/p&gt;&lt;p&gt;Ist man mit einem guten Papierschneider gesegnet, sind bei kleinen Auflagen 0mm kein Problem. Die Fehlerhäufigkeit steigt allerdings mit der Bogendicke des Papiers, da Teile beim Schneiden verrutschen.&lt;/p&gt;&lt;p&gt;Wer es weniger akribisch haben will, arbeitet mit der Schere und nimmt Ungenauigkeiten als Ausdruck echter Handarbeit in Kauf.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QSpinBox" name="spinBox_AnzahlLabels">
          <property name="toolTip">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Hier wird die &lt;span style=&quot; font-weight:600;&quot;&gt;Anzahl &lt;/span&gt;der Etiketten auf einem Etikettenbogen festgelegt. Es gilt die Devise möglichst effizient und ressourcenschonend zu arbeiten.&lt;/p&gt;&lt;p&gt;Üblicherweise liegt im Hobbybereich ein DIN-A4-Bogen vor. Kommen andere Formate oder gar Rollenpapier zum Einsatz, sind größere Änderungen am Layout nötig.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
          <property name="maximum">
           <number>9999999</number>
          </property>
          <property name="value">
           <number>25</number>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLabel" name="label_5">
          <property name="minimumSize">
           <size>
            <width>30</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>30</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>mm</string>
          </property>
         </widget>
        </item>
        <item row="5" column="1">
         <widget class="QLabel" name="label_6">
          <property name="minimumSize">
           <size>
            <width>30</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>30</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>mm</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0" colspan="2">
         <widget class="QLabel" name="label_77">
          <property name="text">
           <string>Breite Label</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0" colspan="2">
         <widget class="QLabel" name="label_49">
          <property name="text">
           <string>Anzahl Labels</string>
          </property>
         </widget>
        </item>
        <item row="4" column="0" colspan="2">
         <widget class="QLabel" name="label_76">
          <property name="text">
           <string>Abstand zwischen den Labels</string>
          </property>
         </widget>
        </item>
        <item row="6" column="0" colspan="2">
         <widget class="QGroupBox" name="groupBox_21">
          <property name="title">
           <string>Seitenränder</string>
          </property>
          <layout class="QGridLayout" name="gridLayout">
           <item row="5" column="1">
            <widget class="QSpinBox" name="spinBox_FLabel_RandRechts">
             <property name="toolTip">
              <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Hier wird der Abstand zum Papierrand &lt;span style=&quot; font-weight:600;&quot;&gt;rechts &lt;/span&gt;festgelegt.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
             </property>
             <property name="maximum">
              <number>210</number>
             </property>
             <property name="value">
              <number>5</number>
             </property>
            </widget>
           </item>
           <item row="7" column="1">
            <widget class="QSpinBox" name="spinBox_FLabel_RandUnten">
             <property name="toolTip">
              <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Hier wird der Abstand zum Papierrand &lt;span style=&quot; font-weight:600;&quot;&gt;unten &lt;/span&gt;festgelegt.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
             </property>
             <property name="maximum">
              <number>290</number>
             </property>
             <property name="value">
              <number>15</number>
             </property>
            </widget>
           </item>
           <item row="3" column="2">
            <widget class="QLabel" name="label_2">
             <property name="minimumSize">
              <size>
               <width>30</width>
               <height>0</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>30</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="text">
              <string>mm</string>
             </property>
            </widget>
           </item>
           <item row="3" column="1">
            <widget class="QSpinBox" name="spinBox_FLabel_RandLinks">
             <property name="toolTip">
              <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Hier wird der Abstand zum Papierrand &lt;span style=&quot; font-weight:600;&quot;&gt;links &lt;/span&gt;festgelegt.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
             </property>
             <property name="maximum">
              <number>210</number>
             </property>
             <property name="value">
              <number>5</number>
             </property>
            </widget>
           </item>
           <item row="5" column="2">
            <widget class="QLabel" name="label_3">
             <property name="minimumSize">
              <size>
               <width>30</width>
               <height>0</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>30</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="text">
              <string>mm</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QSpinBox" name="spinBox_FLabel_RandOben">
             <property name="toolTip">
              <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Hier wird der Abstand zum Papierrand &lt;span style=&quot; font-weight:600;&quot;&gt;oben &lt;/span&gt;festgelegt.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
             </property>
             <property name="maximum">
              <number>297</number>
             </property>
             <property name="value">
              <number>10</number>
             </property>
            </widget>
           </item>
           <item row="1" column="2">
            <widget class="QLabel" name="label">
             <property name="minimumSize">
              <size>
               <width>30</width>
               <height>0</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>30</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="text">
              <string>mm</string>
             </property>
            </widget>
           </item>
           <item row="7" column="2">
            <widget class="QLabel" name="label_4">
             <property name="minimumSize">
              <size>
               <width>30</width>
               <height>0</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>30</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="text">
              <string>mm</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="label_72">
             <property name="text">
              <string>Oben</string>
             </property>
            </widget>
           </item>
           <item row="3" column="0">
            <widget class="QLabel" name="label_73">
             <property name="text">
              <string>Links</string>
             </property>
            </widget>
           </item>
           <item row="5" column="0">
            <widget class="QLabel" name="label_74">
             <property name="text">
              <string>Rechts</string>
             </property>
            </widget>
           </item>
           <item row="7" column="0">
            <widget class="QLabel" name="label_75">
             <property name="text">
              <string>Unten</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item row="8" column="0" colspan="2">
         <widget class="QPushButton" name="btnToPdf">
          <property name="toolTip">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Mit Klick wird hiermit die Banderole oder das Etikett als &lt;span style=&quot; font-weight:600;&quot;&gt;druckbares PDF &lt;/span&gt;gespeichert.&lt;/p&gt;&lt;p&gt;Direktes Drucken ist im Kleinen Brauhelfer (leider) nicht möglich.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
          <property name="text">
           <string>PDF speichern</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ComboBox</class>
   <extends>QComboBox</extends>
   <header>widgets/combobox.h</header>
  </customwidget>
  <customwidget>
   <class>SvgView</class>
   <extends>QGraphicsView</extends>
   <header>widgets/svgview.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>cbAuswahl</tabstop>
  <tabstop>btnLoeschen</tabstop>
  <tabstop>viewSvg</tabstop>
  <tabstop>cbEditMode</tabstop>
  <tabstop>cbTagsErsetzen</tabstop>
  <tabstop>btnSaveTemplate</tabstop>
  <tabstop>tbTemplate</tabstop>
  <tabstop>tbTags</tabstop>
  <tabstop>spinBox_BreiteLabel</tabstop>
  <tabstop>spinBox_AnzahlLabels</tabstop>
  <tabstop>spinBox_AbstandLabel</tabstop>
  <tabstop>spinBox_FLabel_RandOben</tabstop>
  <tabstop>spinBox_FLabel_RandLinks</tabstop>
  <tabstop>spinBox_FLabel_RandRechts</tabstop>
  <tabstop>spinBox_FLabel_RandUnten</tabstop>
  <tabstop>btnToPdf</tabstop>
 </tabstops>
 <resources>
  <include location="images.qrc"/>
 </resources>
 <connections/>
</ui>
